app/javascript/dashboard/routes/dashboard/settings/inbox/channels/Twilio.vue

```diff
@@ -101,11 +83,17 @@ export default {
     PageHeader,
```

Comment:
The `PageHeader` component has been registered but not used in the component

Reply:
Ok I've updated the PR

Reply:
@muhsin-k I've removed PageHEader from Twilio.vue